spec/lib/diaspora/fetcher/public_spec.rb

```diff
@@ -179,7 +179,7 @@
       end
 
       it "returns true, if the user is remote and hasn't been fetched" do
-        person = FactoryGirl.create(:person, {:diaspora_handle => 'neo@theone.net'})
+        person = FactoryBot.create(:person, {:diaspora_handle => 'neo@theone.net'})
```

Comment:
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.